src/mbgl/text/glyph_pbf.cpp

```diff
@@ -94,12 +92,12 @@ void parseGlyphPBF(GlyphSet& glyphSet, const GlyphRange& glyphRange, const std::
                 glyph.bitmap = AlphaImage(size, reinterpret_cast<const uint8_t*>(glyphData.data()), glyphData.size());
             }
 
-            glyphSet.insert(glyph.id, std::move(glyph));
+            result.push_back(std::move(glyph));
```

Comment:
I doubt that this is a very hot path, but in the rest of the code we try to avoid unnecessarily moving around the glyph bitmap, and growing the vector will cause a few copies. Is it worth trying to avoid that with a `reserve` (we can guess that 256 is a good size for the vector)?